crates/jrsonnet-stdlib/src/arrays.rs

```diff
@@ -253,3 +253,29 @@ pub fn builtin_avg(arr: Vec<f64>, onEmpty: Option<Thunk<Val>>) -> Result<Val> {
 	}
 	Ok(Val::Num(arr.iter().sum::<f64>() / (arr.len() as f64)))
 }
+
+#[builtin]
+pub fn builtin_remove_at(
+	arr: ArrValue,
+	index: usize,
+) -> Result<ArrValue> {
+	let newArrLeft = arr.clone().slice(None, Some(index), None);
+	let newArrRight = arr.clone().slice(Some(index + 1), None, None);
+	return Ok(ArrValue::extended(
+		newArrLeft.unwrap_or(ArrValue::empty()),
+		newArrRight.unwrap_or(ArrValue::empty()))
+	);
+}
+
+#[builtin]
+pub fn builtin_remove(arr: ArrValue, elem: Val) -> Result<ArrValue> {
+	let index = arr.iter().position(
+		|val| equals(&val.unwrap(), &elem).unwrap()
```

Comment:
First unwrap will crash on the following input:
`std.remove([error "1"], 1)`
Because this element can't be evaluated.

The second unwrap will crash on
`std.remove([{a: error "1"}], {a: 1})`
Because field `a` can't be compared, due to evaluation error.

Reply:
good catch, latest change should fix this issue

Reply:
Yep, that works